```diff
{% raw %}--- a/src/components/AlbumDetails.js
+++ b/src/components/AlbumDetails.js
@@ -2,6 +2,7 @@ import { API } from 'aws-amplify';
 import React, { useEffect, useState } from 'react';
 import { Dimmer, Header, Loader, Message, Segment } from 'semantic-ui-react';
 
+import DeleteAlbum from './DeleteAlbum';
 import PhotoList from './PhotoList';
 import { getAlbum } from '../graphql/queries';
 import useAuth from '../useAuth';
@@ -20,6 +21,10 @@ function AlbumDetails({ album }) {
       <Segment data-test="album-content" tertiary>
         <PhotoList photos={photos} />
       </Segment>
+
+      <Segment data-test="album-footer" clearing secondary>
+        <DeleteAlbum album={album} />
+      </Segment>
     </Segment.Group>
   );
 }{% endraw %}
```